config.toml

```diff
@@ -107,6 +107,11 @@ version = "0.0"
 # point people to the main doc site.
 url_latest_version = "/"
 
+[[params.versions]]
```

Comment:
Also can we can the `version_menu = "Releases"` to `version_menu = "Docs"` ?

Reply:
Done